### PR TITLE
virus assignment contribution 

### PR DESCRIPTION
**Pull request by alstonlo**

### CODE REVIEW

File: virus/hespivirus_rivalense.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
+<div align="center">
+
+![](imgs/hespivirus_rivalense/cover.png)
```
Comment (ababaian) [inline virus/hespivirus_rivalense.md:3]: Add figure titles

File: virus/hespivirus_rivalense.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+- An alpha-ketoglutarate-dependent dioxygenase AlkB-like domain (IPR027450) in ORF1.
+- A RNA virus helicase core domain (IPR027351) in ORF1.
+- An RdRp from *Alsuviricetes* (IPR001788) in ORF1.
+- A transmembrane helix (TMh) that is part of a membrane-bound protein in ORF2.
+- A Tobacco mosaic virus-like coat protein (TMVC) (IPR001337) in ORF3.
+
+We are unable to find any elements encoding for non-coding RNAs in `Hriv_contig`, using Infernal cmscan 1.1.2. The
+following figure summarizes the contig-wide coverage, ORFs, predicted domains, and RdRp A, B, C motifs.
+
+![](imgs/hespivirus_rivalense/contig.png)
```
Comment (ababaian) [inline virus/hespivirus_rivalense.md:125]: Add a Figure Title, and a proper Figure Legend to your figures here. Below the image it should say *Figure X* and give a brief description. Please do add a short description of the methods you used for this figure. I really like it and would like to use this one as an example in the future, so document `bowtie2` if that's what you used etc..

File: virus/hespivirus_rivalense.md [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+We use the putative RdRp sequence from above. For BLASTP [23], we search against the nr database restricted to RNA
+viruses (taxid:2559587), and we set the maximum number target sequences to 1000. Filtering the candidates with >80%
+query coverage and >40% identity leaves 191 sequences (including the *H. rivalense* RdRp). We further remove entries
+with duplicate scientific names or named "ssRNA positive-strand virus sp." and "Virgaviridae sp.", tie-breaking by E
+score. On the remaining 81 sequences, we perform a multiple sequence alignment (MSA) using MUSCLE [24] and visualize the
+result below using Jalview [25]. For visual clarity, we have trimmed the MSA to the *H. rivalense* RdRp palmprint and
+included only the top 19 BLASTP hits. The A, B, and C motifs of the *H. rivalense* palmprint are annotated in the last
+row, and by inspection, we can see that they are well-conserved.
+
+![](imgs/hespivirus_rivalense/msa.png)
```
Comment (ababaian) [inline virus/hespivirus_rivalense.md:151]: Please include the FASTA MSA data file and .tree as supporting information for this figure. You can store it in the `imgs/hespivirus_rivalense/` folder along with image data.

**Comment by alstonlo** [comment]
Thanks! 

- I have added the legends to all figures. 
- The raw MSA and IQTree files have also been uploaded and linked to within the report.

**Review by ababaian — APPROVED** [review]

**Pull request merged**